framework_tvb/tvb/adapters/simulator/monitor_forms.py

```diff
@@ -32,9 +32,11 @@
 from tvb.core.adapters.abcadapter import ABCAdapter
 from tvb.core.entities.file.simulator.view_model import *
 from tvb.core.entities.filters.chain import FilterChain
+from tvb.datatypes.projections import ProjectionsType
+from tvb.adapters.simulator.equation_forms import get_ui_name_to_monitor_equation_dict, HRFKernelEquation
 from tvb.core.neotraits.forms import Form, ScalarField, ArrayField, MultiSelectField, SelectField, \
     TraitDataTypeSelectField
-from tvb.datatypes.projections import ProjectionsType
+from tvb.basic.neotraits.api import List
```

Comment:
These changed imports seem odd here. Do we need this file changed?

Reply:
Yes, we need the _**List**_ import because it seems that in master this import does not exist but it is used. From what I can see there were some conflicts in the branch where this import was added and at rebase this import was missed.

Reply:
oK. i fix this import in master, as it is relevant for others also.